### DISCUSSION TITLE
Forbidden // inside tag – so how to make a comment?

### DISCUSSION THREAD

**Discussion by dakur** [nette/latte#281] [Q&A] [answered]
Hi,

I use multi-line tags for rich translation with HTML and I sometimes need to comment some of the values passed to translator:

```latte
{_ 'numberOfSectionsRequestedToRead', [
	'number' => 0, // default value is rendered with JS
	'overallCounter' => Nette\Utils\Html::el('span', ['id' => 'overallCounter', 'class' => 'overallCounter']),
	'checkmark' => Nette\Utils\Html::el('span', ['class' => 'numberOfSectionsRequestedToRead']),
]}
```

Previously, commenting with `//` was allowed, but now [it is gone](https://github.com/nette/latte/commit/6dc0b9f89aeccc91ff45d9ed1be7829e0e4ff3a6#diff-a25f68211abf011a3ce571e374e3fafa2d19ee0c52d23c86503f60b1211773efR215-R216). Commenting with `{* *}` also does not work as it would be tag inside of tag.

How can I comment individual lines right now?

Thank you.

**Comment by JanTvrdik** [comment, marked as answer]
`/* ... */` comment should work

**Reply by dakur** [reply]
Yes, that works, thank you very much!
